.github/workflows/update-longhorn.yaml

```diff
@@ -12,10 +12,6 @@ jobs:
       with:
         fetch-depth: 0
 
-    - uses: actions/setup-go@v3
-      with:
-        go-version: 1.17
-
```

Comment:
longhorn uses this:

https://github.com/replicatedhq/kURL/blob/dd0ed9d2b0a3500b7d6e60419339f1d3375e1fac/addons/longhorn/template/generate.sh#L14-L23

Reply:
ah. got it